source/Irrlicht/COpenCLHandler.h

```diff
@@ -211,8 +211,10 @@ class COpenCLHandler
 
         static const SOpenCLPlatformInfo& getPlatformInfo(const size_t& ix) {return platformInformation[ix];}
 
+        static cl_context getOpenCLContext() {return openCLContext;}
+
 #ifdef _IRR_COMPILE_WITH_OPENGL_
-        static bool getCLDeviceFromGLContext(cl_device_id& outDevice,
```

Comment:
we need both functions, for example when the OpenCL device has already been created externally and we need to see if its the right one for sharing with the OpenGL GPU.